src/server/api/routers/promptVariants.router.ts

```diff
@@ -51,7 +52,9 @@ export const promptVariantsRouter = createTRPCRouter({
           id: true,
         },
         where: {
-          modelOutput: {
+          modelResponse: {
+            outdated: false,
+            output: { not: Prisma.AnyNull },
```

Comment:
I think this is probably right but did you run an eval or two and make sure the results look reasonable?

Reply:
Evals appear to be broken already, so I'll debug that in another PR.